tests/golos.referral_tests.cpp

```diff
@@ -16,14 +16,15 @@ class golos_referral_tester : public golos_tester {
     golos_referral_tester()
         : golos_tester(cfg::referral_name)
         , _sym(3, "GLS")
+        , _sym_vest(6, "GLS")
         , post( {this, cfg::publish_name, _sym} )
-        , vest({this, cfg::vesting_name, _sym})
+        , vest({this, cfg::vesting_name, _sym_vest})
         , referral( {this, cfg::referral_name} )
         , token({this, cfg::token_name, _sym})
         , _users{N(sania), N(pasha), N(tania), N(vania), N(issuer)}
     {
         create_accounts({N(sania), N(pasha), N(tania), N(vania), N(issuer), _code,
-                        cfg::publish_name, cfg::token_name, cfg::emission_name, cfg::vesting_name, cfg::social_name });
+                        cfg::publish_name, cfg::token_name, cfg::emission_name, cfg::vesting_name, cfg::social_name, cfg::control_name });
```

Comment:
In which case do you need `cfg::control_name` account? Don't create account if it unused.

Reply:
because when creating a vesting, a default account is specified for notification cfg::control_name